tests/unit/test_action_handler.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+from unittest.mock import Mock
+
+from openslides_backend.action.action_handler import ActionHandler
+from openslides_backend.action.util.typing import Payload
+
+"""
+ACHTUNG !!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+Angefangen zum PullRequest #424 Adjust WR to per-action-base with locked fields
+Joshau sieht in als Unittest, ist aber eher ein system-Test, da er
+mindestens den Datastore-Service braucht, um seine WriteRequests zu erzeugen
+
+"""
```

Comment:
Why is this in this PR? Also, please write comments always in english to keep it uniform.

Reply:
> Why is this in this PR? Also, please write comments always in english to keep it uniform.
Sorry, this has been a remark for myself to not commit this file anywhere ;-) => deleted